app/src/main/java/com/github/sdp/tarjetakuna/utils/ExportCollection.kt

```diff
@@ -0,0 +1,121 @@
+package com.github.sdp.tarjetakuna.utils

```

Comment:
better in a package ui.settings.export...

Reply:
See comment above